### PR TITLE
ENS symbol renders on proper networks

### LINKED ISSUE #1554: Non-ENS names should not show the green check mark
The green check is just for ENS name validation, we shouldn't show it unless it's an ENS name.

screen shot 2017-06-06 at 2 05 10 pm

### PR DESCRIPTION
**Pull request by 2-am-zzz**
Messed up a bit by #1545, but fixes this by adding a few more conditionals. Fixes #1554

### CODE REVIEW

File: ui/app/components/ens-input.js [hunk 1]
```diff
@@ -73,6 +72,7 @@ EnsInput.prototype.render = function () {
 EnsInput.prototype.componentDidMount = function () {
   const network = this.props.network
   const networkHasEnsSupport = getNetworkEnsSupport(network)
+  this.setState({ ensResolution: '0x0000000000000000000000000000000000000000' })
```
Comment (flyswatter) [inline ui/app/components/ens-input.js]: When repeatedly using a long variable in a file, it's a good place to define a `const` at the top of the file. Avoid having to make sure you typed all those zeroes right each time.

File: ui/app/components/ens-input.js [hunk 2]
```diff
@@ -85,14 +85,6 @@ EnsInput.prototype.lookupEnsName = function () {
   const recipient = document.querySelector('input[name="address"]').value
   const { ensResolution } = this.state
 
-  if (!this.ens) {
-    return this.setState({
-      loadingEns: false,
-      ensFailure: true,
-      hoverText: 'ENS is not supported on your current network.',
-    })
-  }
```
Comment (flyswatter) [inline ui/app/components/ens-input.js]: What does removing this do, when trying an `.eth` address on a non-eth network? Just show "no address for this name"?

**Review by danfinlay — APPROVED** [review]
Looks good, a couple minor style notes.

File: ui/app/components/ens-input.js (hunk 1 above)
Comment (kumavis) [inline ui/app/components/ens-input.js]: commonly referred to as the "zero address" or "null address"

File: ui/app/components/ens-input.js (hunk 2 above)
Comment (Zanibas) [inline ui/app/components/ens-input.js]: We don't ever hit this part of the code if the network does not support ENS, due to the conditional on the `mount` function.

**Pull request merged**